src/App.js

```diff
@@ -3,13 +3,17 @@ import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
 function App() {
-  const [newTodo, setNewTodo] = React.useState("");
+  const [todoList, setTodoList] = React.useState([]);
+
+  function addTodo(newTodo) {
+    setTodoList((obj) => [...obj, newTodo]);
```

Comment:
Nitpick (doesn't require changes):
Great job on using the state setter callback function!!! One minor call-out here is that the `obj` argument variable name is a bit ambiguous. A common argument naming convention you'll see is `prev` followed by the name of state variable: `prevTodoList`. This makes it a bit clearer that you're spreading out the previous state and appending a new todo:
```js
setTodoList((prevTodoList) => [...prevTodoList, newTodo]);
```